Feature/sport 335 v1 connection to v2

RouxAntoine opened the pull request.

RouxAntoine committed:
feature-sport-335: retrieve member-hub v1 connection

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/Decathlon/hub-decathlon?pullRequest=193) <br/>All committers have signed the CLA.

kevforget committed:
Extracted databases reading into dedicated file + connections and user dto

feature-sport-335: refactor and map hub v1 connection to domain struct

feature-sport-335: add support for token expiration with date format

fix: polar partner_name to polarflow

fix: garmin partner_name to garminhealth

kevforget committed:
Added postgres connexion import

RouxAntoine commented:
Suite à erreur : commit 5202158 supprimé et mot de passe preprod changé, env preprod redéploy

RouxAntoine committed:
feature: refactoring insert into hub v2
